homeworks/01-bubble-sort/src/components/Column.tsx

```diff
@@ -0,0 +1,13 @@
+export type ColumnProps = {
+    el: number,

```

Comment:
What is `el`? It's a number and it goes to height style... Extremely unreadable naming...